src/ignitus-StudentLogin/actions.js

```diff
@@ -7,14 +7,14 @@ import _ from 'lodash';
 
 export const logInRequest = (email, password) => {
   if (!_.isString(email))
-    throw new Error(`email must be number: ` + email);
+    throw new Error(`email must be a string: ` + email);
   if (!_.isString(password))
-    throw new Error(`password must be number: ` + password);
+    throw new Error(`password must be a string: ` + password);
```

Comment:
Lets just not check about string password. We will have to allow strict password.

Reply:
Do you know what the requirements are? Like length and complexity.

Reply:
For login, I would suggest to ignore other restrictions. Just shouldn't be blank.

Reply:
Agree with you. 
Could you just remove them and add 
1# email should be valid email
2# password should not be empty

Also, remove this checking of form filling. It should check individually and prompt.